src/sentry/db/models/manager.py

```diff
@@ -443,25 +452,27 @@ def get_many_from_cache(self, values, key="pk"):
 
         return final_results
 
-    def create_or_update(self, **kwargs):
-        return create_or_update(self.model, **kwargs)
+    def create_or_update(self, **kwargs: Any) -> Tuple[Any, bool]:
+        model, created = create_or_update(self.model, **kwargs)
```

Comment:
how is this explicitly typed?

Reply:
The original line:
```python
def create_or_update(self, **kwargs: Any) -> Tuple[Any, bool]:
    return create_or_update(self.model, **kwargs)
```
gets this error:
```
Returning Any from function declared to return "Tuple[Any, bool]"
```
So explicitly expanding out the tuple:
```python
def create_or_update(self, **kwargs: Any) -> Tuple[Any, bool]:
    model, created = create_or_update(self.model, **kwargs)
    return model, created
```
satisfies mypy.

Reply:
fwiw this is technically a runtime behavior change because now it crashes on e.g. `None` while previously it didn't (and downstream code may have handled this)

In such situations I would propose using `cast` or using `# type: ignore`